hoodie-common/src/main/java/com/uber/hoodie/common/table/log/HoodieLogFormatVersion.java

```diff
@@ -80,18 +80,15 @@ public boolean hasHeader() {
 
   @Override
   public boolean hasFooter() {
-    switch (super.getVersion()) {
-      case DEFAULT_VERSION:
-        return false;
-      case 1:
-        return true;
-      default:
-        return false;
-    }
+    return getResultByVersion();
   }
 
   @Override
   public boolean hasLogBlockLength() {
+    return getResultByVersion();
+  }
+
+  private boolean getResultByVersion() {
```

Comment:
Good cleanup.  Can you rename this to hasLogFooterAndBlockLength() ?

Reply:
ok, Thank you for reviewing the code.